website/client/js/controllers/inviteToGroupCtrl.js

```diff
@@ -44,8 +44,13 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', '$rootScope', 'User', 'Group
 
       Groups.Group.invite($scope.group._id, invitationDetails)
         .then(function() {
-          Notification.text(window.env.t('invitationsSent'));
-          _resetInvitees();
+           if ((inviteMethod === 'email' && invitationDetails.emails.length > 1) || (inviteMethod === 'uuid' && invitationDetails.uuids.length > 1)) { // test if it's inviting more than 1 emails or uuid
+                Notification.text(window.env.t('invitationsSent'));
+            }
+            else {
+                Notification.text(window.env.t('invitationSent'));
+            }
+            _resetInvitees();
```

Comment:
@Ozzystrasza Take a look at the existing tests in [inviteToGroupCtrlSpec.js](https://github.com/Ozzystrasza/habitrpg/blob/934aa2f04e7d690f734689844f217141ecddc1b7/test/spec/controllers/inviteToGroupCtrlSpec.js).